src/components/FileVideo.tsx

```diff
@@ -76,13 +80,22 @@ const FileVideo: FC<FileVideoProps> = ({
     if (!isPlayerView) {
       navigation.setOptions({
         headerRight: () => (
-          <View>
+          <View
+            style={
+              isDownloading
+                ? styles.headerButtonsDownloadingState
+                : styles.headerButtons
+            }
+          >
+            <FileHeaderButton
+              name="chat"
+              handler={() => handleOpenChat(navigation, item)}
+              testID={CHAT_BUTTON_HEADER}
+            />
             {isDownloading ? (
-              <View style={styles.headerButtonsDownloadingState}>
-                <FileHeaderButton disabled={true} name="cloud-download" />
-              </View>
+              <FileHeaderButton disabled={true} name="cloud-download" />
             ) : (
-              <View style={styles.headerButtons}>
+              <>
```

Comment:
Remove

Reply:
This is still valid I think.

Reply:
![simulator_screenshot_F94EC3D1-FE79-4CDE-AEEE-3E1DBF8CDC67](https://github.com/graasp/graasp-mobile-builder/assets/67112345/5f1ae90f-d69d-4e26-9c2c-5087ce9b7028)
Buttons are aligned in column by default